site_root/input/resources/profiles/StructureDefinition-kfdrc-condition-no-phi.json

```diff
@@ -0,0 +1,55 @@
+{
+    "resourceType": "StructureDefinition",
+    "id": "kfdrc-condition-no-phi",
+    "url": "http://fhir.kids-first.io/StructureDefinition/kfdrc-condition-no-phi",
+    "version": "0.1.0",
+    "name": "kfdrc-condition-no-phi",
+    "title": "KFDRC Condition Without PHI",
+    "status": "draft",
+    "fhirVersion": "4.0.0",
+    "kind": "resource",
+    "abstract": false,
+    "type": "Condition",
+    "baseDefinition": "http://fhir.kids-first.io/StructureDefinition/kfdrc-condition",
+    "derivation": "constraint",
+    "differential": {
+      "element": [
+        {
+            "id": "Condition.onsetDateTime",
+            "path": "Condition.onsetDateTime",
+            "max": "0"
+        },
+        {
+            "id": "Condition.onsetPeriod",
+            "path": "Condition.onsetPeriod",
+            "max": "0"
+        },
+        {
+            "id": "Condition.onsetString",
```

Comment:
This doesn't seem like a PHI attribute?

Reply:
I wanted to prevent someone putting a date in as string. wasn't sure what the value of string is if age is available.

Reply:
See this as well: https://kidsfirst-team.slack.com/archives/CDXG6FKDE/p1586891473118700?thread_ts=1586889504.114000&cid=CDXG6FKDE